doc/tutorials/bare_flame_game.md

```diff
@@ -4,8 +4,8 @@ This tutorial assumes that you have basic familiarity with using the command lin
 following programs on your computer (all of them are free):
 
 - [Flutter], version 2.10.0 or above;
-- [Android Studio], or any other IDE, for example [Visual Studio Code];
-- [git] (optional), in order to save your project on GitHub.
+- [Android Studio], or any other IDE, for example, Visual Studio Code];
```

Comment:
This extra comma makes the sentence harder to read, and one of the links was accidentally removed

Reply:
Fixed.